{"name": "Due date accepts only date value on MM/dd/yyy -- @1.3 ", "status": "failed", "statusDetails": {"message": "AssertionError: Test Failed! Should not have saved an empty subtask\n", "trace": "  File \"/Users/sudippudasaini/Apps/behaveAvenueCode/venv/lib/python3.11/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Users/sudippudasaini/Apps/behaveAvenueCode/venv/lib/python3.11/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/subtasks.py\", line 76, in verify_empty_subtask\n    assert False, \"Test Failed! Should not have saved an empty subtask\"\n"}, "steps": [{"name": "Given I am logged in with \"sudip_pudasaini@hotmail.com\" and password \"assessmentqa\"", "status": "passed", "start": 1674801497104, "stop": 1674801500495}, {"name": "And I land on my tasks page", "status": "passed", "start": 1674801500495, "stop": 1674801500514}, {"name": "When a task is added", "status": "passed", "start": 1674801500514, "stop": 1674801500536}, {"name": "When I click on manage subtasks button", "status": "passed", "start": 1674801500536, "stop": 1674801505810}, {"name": "And I wipe out the default date value", "status": "passed", "start": 1674801505811, "stop": 1674801505886}, {"name": "And enter \"123123123123\" on due date field", "status": "passed", "start": 1674801505886, "stop": 1674801505957}, {"name": "And I click on add subtask button", "status": "passed", "start": 1674801505957, "stop": 1674801506044}, {"name": "Then I should not see an empty subtask getting added", "status": "failed", "statusDetails": {"message": "AssertionError: Test Failed! Should not have saved an empty subtask\n", "trace": "  File \"/Users/sudippudasaini/Apps/behaveAvenueCode/venv/lib/python3.11/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Users/sudippudasaini/Apps/behaveAvenueCode/venv/lib/python3.11/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/subtasks.py\", line 76, in verify_empty_subtask\n    assert False, \"Test Failed! Should not have saved an empty subtask\"\n"}, "start": 1674801506044, "stop": 1674801506111}], "parameters": [{"name": "value", "value": "123123123123"}], "start": 1674801487728, "stop": 1674801506258, "uuid": "af736aae-c290-40c4-bf14-745a7fc384b3", "historyId": "c816a8052f077d3e07080aed8113923a", "fullName": "Manage subtasks: Due date accepts only date value on MM/dd/yyy", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Manage subtasks"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}